plugin/cmd/kube-scheduler/app/server.go

```diff
@@ -80,13 +81,22 @@ func (s *SchedulerServer) Run(_ []string) error {
 	}
 
 	go func() {
+		mux := http.NewServeMux()
 		if s.EnableProfiling {
```

Comment:
Should we just enable by default?  Then we can capture and report data in e2e.  I know @vmarmol thought so.